next/pages/overenie-identity.tsx

```diff
@@ -45,11 +47,13 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
 
 const IdentityVerificationPage = ({ page }: AsyncServerProps<typeof getServerSideProps>) => {
   const { t } = useTranslation('account')
+  const [lastIco, setLastIco] = useState<string | undefined>()
   const [lastRc, setLastRc] = useState('')
   const [lastIdCard, setLastIdCard] = useState('')
 
   const [identityVerificationError, setIdentityVerificationError] = useState<Error | null>(null)
-  const { isAuthenticated, tierStatus } = useServerSideAuth()
+  // TODO fix is legal entity
```

Comment:
forgotten or refering to 

> // helper, since we usually determine what to display this way, slightly convoluted because of ts rules & undefined vs boolean

?

Reply:
forgotten